spec/views/accounts/connected_accounts/show.html.erb_spec.rb

```diff
@@ -29,6 +29,7 @@
 
     it 'renders' do
       expect { render }.to_not raise_error
+      expect(rendered).to match '</time>'
```

Comment:
sometimes another check I like for HTML escaping is to do:
```suggestion
      expect(rendered).to match '</time>'
      expect(rendered).to_not include('&lt;')
```
but otherwise same same